llvm/test/lab3/vasilev_ivan/test2.mir

```diff
@@ -0,0 +1,560 @@
+# RUN: llc -mtriple x86_64-unknown-linux-gnu --load=%llvmshlibdir/X86VasilevMulAddPass%shlibext -run-pass=x86-Vasilevmuladd  %s -o - | FileCheck %s
+
+--- |
+  ; ModuleID = '/home/adandroll/lab3/llvm-nnsu-2024/llvm/test/lab3/vasilev_ivan/test2.ll'
+  source_filename = "/home/adandroll/lab3/llvm-nnsu-2024/llvm/test/lab3/vasilev_ivan/test2.cpp"
+  target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+  target triple = "x86_64-pc-linux-gnu"
+  
+  ; Function Attrs: mustprogress noinline nounwind optnone uwtable
+  define dso_local noundef <2 x double> @_Z12muladd_test1Dv2_dS_S_(<2 x double> noundef %0, <2 x double> noundef %1, <2 x double> noundef %2) #0 {
```

Comment:
Please remove duplicated test cases.

Reply:
yes, sorry, I just wanted to make 1 file, so I moved all the tests + added a new one, forgot to delete the previous file with the tests

Reply:
maybe just delete tust.mir?